Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -53,22 +42,33 @@ spec:
 
     resources:
       limits:
-        cpu: "500m"
-        memory: "1024Mi"
+        cpu: "3"
+        memory: "8Gi"
       requests:
-        cpu: "500m"
-        memory: "1024Mi"
+        cpu: "3"
+        memory: "8Gi"
 
-    priorityClassName:
+    priorityClassName: ""
 
     logLevel: NIV_INFO
+
+  security:
+    sssd:
+      sidecar:
+        image: registry.access.redhat.com/rhel7/sssd:latest
+
+        sssdConfigFile:
+          configMap:
+            name: my-nfs-sssd-config
+
+        debugLevel: 6
```

Comment:
Should this debug level be lowered before merge?

Reply:
It's here as an example. I could comment it out, but I think the way non-commented yaml renders is easier to understand. I could set it to `0`, which allows SSSD to use the default.